src/main/screen_textentry.cpp

```diff
@@ -455,14 +490,14 @@ bool KeyboardMouseRender(bool mouse, bool render)
         key_size &= ~1;
     }
 
-    int kb_height = 5*key_size;
-    int kb_width = 12*key_size;
+    int kb_height = s_current_keymap_rows*key_size;
+    int kb_width = s_current_keymap_cols*key_size;
 
     int win_width = kb_width + 20;
     int win_height = kb_height + 20;
 
     int n_prompt_lines = ((s_Prompt.size() + 26) / 27);
-    int n_text_lines = ((Text.size() + 24) / 25);
```

Comment:
I need to update the 25 chars per line constant, based on the number of keyboard columns.